test/test_gipc.py

```diff
@@ -345,7 +344,13 @@ def test_typecheck_kwargs(self):
 
     @mark.skipif('WINDOWS')
     def test_exitcode_previous_to_join(self):
-        p = start_process(lambda: gevent.sleep(SHORTTIME))
+        p = None
+        if multiprocessing.get_start_method() == 'spawn':
+            # Spawn cannot pass lambdas to subprocesses
+            # as the process space is not copied
+            p = start_process(gevent.sleep, (SHORTTIME,))
```

Comment:
I'd be OK with "always" using `p = start_process(gevent.sleep, (SHORTTIME,))` here :)

Reply:
(Thanks for your desire to not change the 'old' test!)

Reply:
I kind of like keeping it, because passing lambdas is still valid for forking, and I imagine someone somewhere is probably doing it.

Reply:
Maybe it should be it's own test, that skips if spawn is enabled.